keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -486,8 +488,8 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
 
       context("deleting a delivery config") {
         before {
-          store()
           storeResources()
+          store()
```

Comment:
The order of calls is important here. If the resources are not persisted before the delivery config, they do not get attached to it. This was never working.

Reply:
Oh wow! Good find! 😬